spec.md

```diff
@@ -782,6 +782,7 @@ For success responses, the following fields are defined:
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
 | description | string | A user-facing message that can be used to tell the user details about the status of the operation. If present, MUST be a non-empty string. |
+| instance_corrupt | boolean | For failed update and deprovisioning operations, this field indicates whether the instance is still usable or not. If the value is `true`, the Service Instance MUST be considered corrupt and the Platform SHOULD NOT allow the creation of new bindings. If the value is `false`, the Service Instance is in an unmodified and usable state. If not specified, the state of the resource is unspecified by this specification. |
```

Comment:
Are there invalid combinations of `instance_corrupt` and `state`? If so we should be explicit about them. Is it valid to see `state: succeeded` and `instance_corrupt: true`?

Can I update a corrupt instance?

Reply:
The `For failed update and deprovisioning operations...` text implies it only applies when state="failed" but I agree we should be more explicit about it.